internal/cmd/docker/push.go

```diff
@@ -129,6 +129,7 @@ func logPushProgress(reader io.ReadCloser) error {
 
 			// Create a progress spinner for statuses that don't have progress details, like 'Preparing'.
 			if msg.Progress == nil || msg.Progress.Total == 0 {
+				//nolint:govet
```

Comment:
`golangci-lint` was updated and introduced a new strict rule that disallows `non-constant format strings`. This line can't be resolved by simply updating the message, so it should be ignored.